src/classes/CRLP_Batch_Base_Skew.cls

```diff
@@ -217,6 +221,8 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
                 parentKeyField = 'ContactId';
             } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.AccountContactSoftCredit) {
                 parentKeyField = 'Contact.AccountId';
+            } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.AccountSoftCredit) {
+                parentKeyField = 'Account__c';
```

Comment:
Not sure this will work in a namespaced context. The rest of the `parentKeyField` values are standard fields. This is the only one referencing a custom field. I feel like it should have the namespace applied, though I'm always a bit unsure of when the namespace is absolutely required for these kind of references. Probably worth testing in a namespaced scratch org to be sure it's working as expected.